misc/json-to-record-converter/src/main/java/io/ballerina/jsonmapper/JsonToRecordMapper.java

```diff
@@ -138,7 +159,10 @@ public static JsonToRecordResponse convert(String jsonString, String recordName,
         Token eofToken = AbstractNodeFactory.createIdentifierToken("");
         ModulePartNode modulePartNode = NodeFactory.createModulePartNode(imports, moduleMembers, eofToken);
         try {
-            response.setCodeBlock(Formatter.format(modulePartNode.syntaxTree()).toSourceCode());
+            FormattingOptions formattingOptions = FormattingOptions.builder()
+                    .setForceFormattingOptions(ForceFormattingOptions.builder()
+                            .setForceFormatRecordFields(forceFormatRecordFields).build()).build();
```

Comment:
Shall we separate out the force formatting options builder, just to be more readable?